Added a Raspberry Pi installation guide (#30)
mysteriumnetwork/docs: docs/user-guide/installation.md

E.g. My publicly visible ip is [78.X.Y.Z] and my laptop's local ip [192.168.1.102], so I port forwarded 78.X.Y.Z:1194 -> 192.168.1.102:1194 in router, which I have access to.

## Running mysterium node on a Raspberry Pi

Mysterium node supports running a service on ARM architecture. As an example of such installation, it can be installed and running on the Raspberry Pi 3 Model B+.

### Preinstallation requirements
Installation of the Mysterium node was tested on the Raspbian operation system, but most of the other systems should be supported too.

Download and install a Raspbian image [https://www.raspberrypi.org/downloads/](https://www.raspberrypi.org/downloads/) to the SD card as described in the official Raspberry Pi documentation [https://www.raspberrypi.org/documentation/installation/installing-images/](https://www.raspberrypi.org/documentation/installation/installing-images/).

> Once you have an installed operating system, make sure that you have an Internet connection, since the Mysterium Node requires it to be running. Both wired and wireless connection supported, but the wired connection should give a more stable and performant connection.

Reviewer: ```suggestion
Once you have an installed operating system, make sure that you have an Internet connection, since the Mysterium Node requires it to run. Both wired and wireless connections are supported, but the wired connection should give a more stable and performant connection.
```

> To support running a Wireguard service it's required to install it separately. Here is a description of how it can be installed: [https://github.com/adrianmihalko/raspberrypiwireguard](https://github.com/adrianmihalko/raspberrypiwireguard).

Reviewer: ```suggestion
To run a Wireguard service you'll need to install Wireguard separately. Here is a description of how it can be installed: [https://github.com/adrianmihalko/raspberrypiwireguard](https://github.com/adrianmihalko/raspberrypiwireguard).
```
